subprojects/libnvme.wrap

```diff
@@ -1,6 +1,6 @@
 [wrap-git]
 url = https://github.com/linux-nvme/libnvme.git
-revision = 568c5b80d1387a1491b09a8254fe039f0675d073
+revision = 662799b5961ac1f4739825181480a32e1effa669
```

Comment:
Could you split the libnvme.wrap update of this patch. In case someone has to backport this patch it will always collide with the wrap update. Yeah  I know it's a bit awkward. Though all will get better in future with libnvme 2.x :)

Reply:
Alright! :)